app/[locale]/dapps/_components/dapps.tsx

```diff
@@ -291,50 +284,18 @@ interface Categories {
   [key: string]: Category
 }
 
-export async function getStaticPaths() {
-  return {
-    paths: LOCALES_CODES.map((locale) => ({ params: { locale } })),
-    fallback: false,
-  }
-}
-
-export const getStaticProps = (async ({ params }) => {
-  const { locale = DEFAULT_LOCALE } = params || {}
-
-  const requiredNamespaces = getRequiredNamespacesForPage("/dapps")
-
-  const contentNotTranslated = !existsNamespace(locale!, requiredNamespaces[2])
-
-  const lastDeployDate = getLastDeployDate()
-  const lastDeployLocaleTimestamp = getLocaleTimestamp(
-    locale as Lang,
-    lastDeployDate
-  )
-
-  const messages = await loadNamespaces(locale, requiredNamespaces)
-
-  return {
-    props: {
-      messages,
-      contentNotTranslated,
-      lastDeployLocaleTimestamp,
-    },
-  }
-}) satisfies GetStaticProps<BasePageProps, Params>
-
 const DappsPage = () => {
   const { t } = useTranslation(["page-dapps", "common"])
-  const { query } = useRouter()
+  const searchParams = useSearchParams()
   const locale = useLocale()
 
   const [selectedCategory, setCategory] = useState<CategoryType>(
     CategoryType.FINANCE
   )
   const explore = useRef<HTMLDivElement>(null)
 
+  const queryParamCategories = searchParams.get("category") || ""
```

Comment:
Current build not working for me locally:
```
Type error: 'searchParams' is possibly 'null'.

  295 |   const explore = useRef<HTMLDivElement>(null)
  296 |
> 297 |   const queryParamCategories = searchParams.get("category") || ""
      |                                ^
  298 |   useEffect(() => {
  299 |     const selectedCategory = queryParamCategories
  300 |       ? (queryParamCategories.split(",")[0] as CategoryType)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

```suggestion
  const queryParamCategories = searchParams?.get("category") || ""
```
...this fixes. Will push a patch, just documenting.